src/connectedvmware/azext_connectedvmware/_params.py

```diff
@@ -351,6 +343,10 @@ def load_arguments(self, _):
             'password', options_list=['--password'],
             help="Username password credentials to use for connecting to the VM.",
         )
+        c.argument(
+            'https_proxy', options_list=['--https-proxy'],
+            help="HTTPS proxy server url for the VM.",
```

Comment:
`--https-proxy` is the default option name, so there is no need to define it repeatedly
```suggestion
            'https_proxy', help="HTTPS proxy server url for the VM.",
```